checkov/common/checks_infra/solvers/attribute_solvers/range_includes_attribute_solver.py

```diff
@@ -12,9 +12,23 @@ def __init__(
             is_jsonpath_check: bool = False
     ) -> None:
         # Convert value to a list if it's not already one to unify handling
-        value = [force_int(v) if isinstance(v, (str, int)) else v for v in
-                 (value if isinstance(value, list) else [value])]
-        super().__init__(resource_types, attribute, value, is_jsonpath_check)
+        value = value if isinstance(value, list) else [value]
+
+        # Process each item in the value list
+        processed_value: List[Any] = []
```

Comment:
few things here:
1. Running code inside constructors is not ideal, is there any reason not to put this block of code inside `get_operation`?
2. Please extract this part to a new function like `_handle_range_values` for readability of the code
3. Update the relevant `Readme` to mention we support ranges

Reply:
1-2 - good callout. Moved.
3 - updated the docs for operators